libs/ui/src/components/Explorer/_queries/useExplorerData.ts

```diff
@@ -69,8 +70,10 @@ export const useExplorerData = ({
         }
     });
 
+    const memoizedData = useMemo(() => (data !== undefined ? _mapping(data, libraryId, availableLangs) : null), [data]);
```

Comment:
**note:** each call to _mapping return a new object, even if data have not changed